src/matrix.jl

```diff
@@ -51,8 +51,8 @@ An optional keyword argument iterator `header` can be passed which will be conve
 `Vector{Symbol}` to be used as the column names. Note that no copy of the `AbstractMatrix`
 is made, but `header` is always stored as freshly allocated object.
 """
-function table(m::AbstractMatrix; header=[Symbol("Column$i") for i = 1:size(m, 2)])
-    symbol_header = [Symbol(h) for h in header]
+function table(m::AbstractMatrix; header=nothing)
```

Comment:
```suggestion
function table(m::AbstractMatrix; header=[Symbol("Column$i") for i = 1:size(m, 2)])
```
it still has to have a reasonable default

Reply:
It hits this code below because `nothing` isn't `Vector{Symbol}`

Reply:
Yeah. could you revert to using `[Symbol("Column$i") for i = 1:size(m, 2)]`